pkg/api/types.go

```diff
@@ -504,20 +506,32 @@ func (l *VolumeList) Set(value string) error {
 	if len(value) == 0 {
 		return errors.New("invalid format, must be source:destination")
 	}
+
+	var pos int
+	if isWindowsRegExp.MatchString(value) {
+		// We need to skip the 'C:\' that prefixes Windows directories, so we only
+		// check for ':' after that prefix.
+		pos = strings.Index(value[3:], ":") + 3
```

Comment:
The documentation of this function says:
```
This function parses the string that contains source:destination pair.
```

Maybe that needs to be revisited. How would it look like on Windows?

```
C:\Users\John\projects\webapp:/destination/in/container
```

?